backend/src/services/upload-file/upload-file.service.ts

```diff
@@ -0,0 +1,33 @@
+import cloudinary from 'cloudinary';
+import DatauriParser from 'datauri/parser';
+import { UploadFileResponse } from '~/common/types/types';
+import { ENV, ResourceType } from '~/common/enums/enums';
+
+interface UploadFileProps {
```

Comment:
```suggestion
type UploadFileProps = {
```
move to `backend/src/services/upload-file/common/types/upload-file-params.type`